webextension/js/popup.js

```diff
@@ -432,27 +440,46 @@ Logic.registerPanel(P_CONTAINER_EDIT, {
     });
   },
 
+  initializeRadioButtons() {
+    const colorRadioTemplate = (containerColor) => {
+      return `<input type="radio" value="${containerColor}" name="container-color" id="edit-container-panel-choose-color-${containerColor}" />
+     <label for="edit-container-panel-choose-color-${containerColor}" class="usercontext-icon choose-color-icon" data-identity-icon="circle" data-identity-color="${containerColor}">`;
+    };
+    const colors = ["blue", "turquoise", "green", "yellow", "orange", "red", "pink", "purple" ];
```

Comment:
Can we pull this definition to the top of `popup.js` like we did with `IDENTITY_COLORS` in `index.js` ?